app/src/main/java/org/stepik/android/adaptive/pdd/ui/adapter/StatsViewPagerAdapter.kt

```diff
@@ -0,0 +1,22 @@
+package org.stepik.android.adaptive.pdd.ui.adapter
+
+import android.content.Context
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentStatePagerAdapter
+import org.stepik.android.adaptive.pdd.R
+import org.stepik.android.adaptive.pdd.ui.fragment.AchievementsFragment
+import org.stepik.android.adaptive.pdd.ui.fragment.ProgressFragment
+import org.stepik.android.adaptive.pdd.ui.fragment.RatingFragment
+
+
+class StatsViewPagerAdapter(fm: FragmentManager, context: Context) : FragmentStatePagerAdapter(fm) {
+    private val fragments = listOf(
+            ProgressFragment::class.java     to context.getString(R.string.progress)!!,
+            AchievementsFragment::class.java to context.getString(R.string.achievements)!!,
+            RatingFragment::class.java       to context.getString(R.string.rating)!!
+    )
+
+    override fun getItem(position: Int) = fragments[position].first.newInstance()!!
```

Comment:
To be honest, do not use `Class#newInstance`. https://stackoverflow.com/a/195328/5222184 
You can use creation block, which return Fragment